src/scripts/main.js

```diff
@@ -1,3 +1,311 @@
+
+      newRowData[tHeadCells[index].innerText.toLowerCase()] = cell.textContent;
+      index++;
+      newRow.append(cell);
+    });
+
+    if (formDataValidation(newRowData)) {
+      tbody.append(newRow);
+      form.reset();
+    }
+  });
+
+  form.append(saveButton);
+  body.append(form);
+}
+
+function createLabel(innerTextForLabel, inputElement, form) {
+  const label = document.createElement('label');
+
+  label.textContent = innerTextForLabel;
+  label.append(inputElement);
+
+  form.append(label);
+
+  return label;
+}
+
+// eslint-disable-next-line no-shadow
+function createInput(name, type = 'text') {
+  const input = document.createElement('input');
+
+  input.required = true;
+  input.setAttribute('data-qa', name);
+  input.name = name;
+  input.type = type;
+
+  if (type === 'number') {
+    input.min = '0';
+  }
+
+  return input;
+}
+
+// eslint-disable-next-line no-shadow
+function createSelect(name, options) {
+  const select = document.createElement('select');
+
+  select.name = name;
+  select.setAttribute('data-qa', name);
+
+  options.forEach((option) => {
+    const optionHTML = document.createElement('option');
+
+    optionHTML.value = option;
+    optionHTML.textContent = option;
+    select.append(optionHTML);
+  });
+
+  return select;
+}
+
+function getSalary(number) {
+  return '$'
+    + parseInt(number).toLocaleString('en-US');
+}
+
+function formDataValidation(newRowData) {
+  let isCorrect = true;
+
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification');
+  notification.setAttribute('data-qa', 'notification');
+  notification.style.textAlign = 'center';
+
+  const notificationTitle = document.createElement('h2');
+
+  notificationTitle.classList.add('title');
+
+  const notificationMessage = document.createElement('p');
+
+  const newEmployeeName = newRowData.name;
+  const newEmployeeAge = newRowData.age;
+
+  const minNameLength = newEmployeeName.length < 4;
+  const minEmployeeAge = parseInt(newEmployeeAge, 10) < adultAge;
+  const maxEmployeeAge = parseInt(newEmployeeAge, 10) > maxAge;
+
+  if (minNameLength || minEmployeeAge || maxEmployeeAge) {
+    if (minNameLength) {
+      notification.classList.add('error');
+      notificationTitle.textContent = 'Name length error';
+
+      notificationMessage.textContent
+        = 'Your name length should be more than 4 symbols';
+    } else if (minEmployeeAge || maxEmployeeAge) {
+      notification.classList.add('error');
+      notificationTitle.textContent = 'Age error';
+      notificationMessage.textContent = 'Your age can\'t be less than 18';
+
+      if (maxEmployeeAge) {
+        notification.classList.add('error');
+        notificationMessage.textContent = 'Your age can\'t be more than 90';
+      }
+    }
+
+    isCorrect = false;
+  } else {
+    notification.classList.add('success');
+    notificationTitle.textContent = 'Success';
+    notificationMessage.textContent = 'Data is correct';
```

Comment:
I'd recommend to move this part of code to a helper function where you will pass sth like: type 'success' or 'error', titleTextContent and messageTextContent. So you will follow DRY principle since you've used the same for notifications above

Reply:
ok